### PR TITLE
Completed Account Settings

### LINKED ISSUE #19: [Feature 16] Logging out
Let's the user log out.

### LINKED ISSUE #22: [Feature 12.1] Changing username
Let's the user change the username.

### LINKED ISSUE #23: [Feature 12.2] Changing password
Let's the user change the password.

### LINKED ISSUE #25: [Feature 12.3] Changing email address
Let's the user change the email address.

### LINKED ISSUE #37: [Feature 12} Changing Account Info
Let's the user change the username, password, email address, and display picture,

### PR DESCRIPTION
**Pull request by CSEndeavour**
Integrated AccountSettings code with UserLogin code. Keeps current user logged in.  Android back button doesn't go back to previous activity except when it's on a pop-up page to change account info.

### CODE REVIEW

**Comment by CSEndeavour** [comment]
The Login placeholder still needs to be removed and one bug is when the android back button is pressed when it's in the RegisterView, it goes to the android home screen instead of going back to the LoginView.

File: app/src/main/res/layout/activity_main2.xml
Comment (ChowdhuryShaswata) [inline app/src/main/res/layout/activity_main2.xml:1]: Rename these to be clearer.

File: app/src/main/java/com/example/notes2text/adapters/AccountSettingsAdaptors/PopUpEmailPresenter.java
Comment (ChowdhuryShaswata) [inline app/src/main/java/com/example/notes2text/adapters/AccountSettingsAdaptors/PopUpEmailPresenter.java:1]: remember to write documentation(docstrings + comments)

**Review by ChowdhuryShaswata — APPROVED** [review]

**Review by JerryLi-617 — APPROVED** [review]
Remember to add documentation for all the codes

**Review by JerryLi-617 — APPROVED** [review]
Remember to add documentation to all codes

**Pull request merged**